config/samples/image-cache/image-cache.yaml

```diff
@@ -0,0 +1,55 @@
+# Sample using Ceph as a glance backend
+# Requires a running Ceph cluster and its `/etc/ceph` files in secret `ceph-conf-files`
+# This can be achieved with the `ceph` target of `install_yamls`
+apiVersion: core.openstack.org/v1beta1
+kind: OpenStackControlPlane
+metadata:
+  name: openstack
+spec:
+  glance:
+    template:
+      databaseInstance: openstack
+      serviceUser: glance
+      containerImage: quay.io/podified-antelope-centos9/openstack-glance-api:current-podified
+      customServiceConfig: |
+        [DEFAULT]
+        enabled_backends = default_backend:rbd
```

Comment:
Since glance uses caching as an middleware, In addition we also need to set below parameter in glance-api.conf to enable the cache ,

[paste_deploy]
flavor = keystone+cachemanagement

If this is not set then I am doubtful that image caching will work.

Reply:
ack and thanks for the feedback, I'm going to update the sample file in a separate patch.